src/components/CommentTime.jsx

```diff
@@ -27,34 +27,35 @@ class CommentTime extends React.Component {
 
     componentDidMount() {
         this._ismounted = true;
-        this.fetchUserNames();
+        this.fetchUpdatedByName();
     }
 
     componentWillUnmount() {
         this._ismounted = false;
     }
 
-    componentDidUpdate() {
-        this.fetchUserNames();
+    componentDidUpdate(prevProps) {
+        if (this.props.accessToken
+            && (prevProps.accessToken !== this.props.accessToken
+                || prevProps.createdBy !== this.props.createdBy
+                || prevProps.updatedBy !== this.props.updatedBy)) {
+            this.fetchUpdatedByName();
+        }
     }
 
-    fetchUserNames() {
+    fetchUpdatedByName() {
         if (!this.state.updatedByName && this.props.updatedBy) {
```

Comment:
This seems incorrect. The `fetchUpdatedByName` should get the data regardless of the `this.state.updatedByName` as the value in the state could (and likely is) old/incorrect.

Reply:
Picked up in #123.